### PR TITLE
 Add validation script for generated code 

### PR DESCRIPTION
**Pull request by kargakis**
@openshift/sig-azure

### CODE REVIEW

**Comment by kargakis** [comment]
Will merge to setup the verification test as part of the build job

File: hack/validate-generated.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+trap "rm -rf ${T}" EXIT
+
+cp -a "${DIR}/.." "${T}/"
+
+(cd "${T}/" && go generate ./...)
+
+
+GENERATED_FILES=(
+	"pkg/addons/bindata.go"
+	"pkg/arm/bindata.go"
```
Comment (mjudeikis) [inline hack/validate-generated.sh]: jsonpath too

File: hack/validate-generated.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```
Comment (jim-minter) [inline hack/validate-generated.sh]: probably should combine hack and tools

Comment (jim-minter) [reply hack/validate-generated.sh]: don't care which stays

**Pull request merged**